.github/workflows/ci.yml

```diff
@@ -108,10 +108,27 @@ jobs:
 
       - name: "Install Chrome"
         uses: browser-actions/setup-chrome@db1b524c26f20a8d1a10f7fc385c92387e2d0477 # v1.7.1
+        id: setup-chrome
+        if: ${{ matrix.nox-session == 'emscripten' }}
+        with:
+          chrome-version: 120
```

Comment:
What version of chrome was in there before the version override? Presumably an old one? We might want to check if it works against chrome 126 since there are some jspi changes there.

Reply:
It will only work against 126 once pyodide 0.27 is properly released, hence I stuck it back on 120 for testing for now, because getting the latest version of chrome breaks everything. I figure it doesn't matter, because the next version of pyodide that includes an upgraded urllib3 will also have the fix for jspi in pyodide (the feature detect thing)

Reply:
fixed with 0.26.3 I think